lib/msal-common/src/error/NetworkError.ts

```diff
@@ -38,7 +38,9 @@ export class NetworkError extends AuthError {
 export function createNetworkError(
     error: AuthError,
     httpStatus?: number,
-    responseHeaders?: Record<string, string>
+    responseHeaders?: Record<string, string>,
+    errorMessage?: string
 ): NetworkError {
+    error.errorMessage = errorMessage || error.errorMessage;
```

Comment:
This should be its own property, don't overwrite the message for the AuthError as that's often what gives developers instructions to resolve

Reply:
Updated. I do not want to introduce a new property, so appended this instead of overwrite. Can you check?